### DISCUSSION TITLE
Astro + PrimeVue template?

### DISCUSSION THREAD

**Discussion by liborjelinek** [primefaces/community#852] [PrimeVue] [answered]
Hi, what happened with Astro + PrimeVue quick start template? I remember it was on the website few months ago. Is Astro still an option?

**Comment by tugcekucukoglu** [comment, marked as answer]
Yes, it is still an option. We renewed all the quickstarts for a while ago and it seems it has been forgotten.

**Reply by calvin2021y** [reply]
any update ?
